homeassistant/components/modbus/strings.json

```diff
@@ -68,5 +68,11 @@
         }
       }
     }
+  },
+  "issues": {
+    "deprecated_close_comm_config": {
+      "title": "CLOSE_COMM_ON_ERROR is deprecated",
```

Comment:
When I check the const.py, the value is actually `close_comm_on_error`. Why do we capitalize it here?

Reply:
ups ! update in 2 seconds.